pkg/graph/resolvers/trb_user_info_dataloader.go

```diff
@@ -0,0 +1,97 @@
+package resolvers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/graph-gophers/dataloader"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// Loaders wrap your data loaders to inject via middleware
+type Loaders struct {
+	UserInfoLoader *dataloader.Loader
+}
+
+type ctxKey string
+
+const (
+	loadersKey = ctxKey("dataloaders")
+)
+
+// UserInfoFetcher reads Users from a database
+type UserInfoFetcher struct {
+	FetchUserInfos func(context.Context, []string) ([]*models.UserInfo, error)
```

Comment:
Have you considered removing the user info Fetcher / or adding it the a parent data loader? It can be a field  on the parent data loader, without needing this additional struct type.

It seems like this works fine for now, but it might be more convenient when additional data loaders get added. The [MINT implementation](https://github.com/CMSgov/mint-app/blob/main/pkg/storage/loaders/data_loaders.go#L11) wraps shared functionality in the data readers object to  provide access to the store.